scripts/extract_policies.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+CHART_DIR=$1
+CHARTS_DIRS=`find $CHART_DIR -type d -exec test -e '{}'/values.yaml \; -print`
+POLICYLIST_FILENAME=policylist.txt
+TMP_IMAGE_FILE=/tmp/$POLICYLIST_FILENAME
+
+if [ -e $POLICYLIST_FILENAME ]; then
+	rm $POLICYLIST_FILENAME
+fi
+for chart in $CHARTS_DIRS; do
+	helm template --values $chart/values.yaml --set recommendedPolicies.enabled=true $chart/ | yq -r "..|.module?" | sort -u | grep -v "null"  > $TMP_IMAGE_FILE
+	sed --in-place '/---/d' $TMP_IMAGE_FILE
+	cat $TMP_IMAGE_FILE | sort -u > $chart/$POLICYLIST_FILENAME
+	# Unified imagelist.txt file. Containing image from all charts.
+	cat $chart/$POLICYLIST_FILENAME >> $POLICYLIST_FILENAME
+done
+# Delete the empty policylist.txt files.
+find $CHART_DIR -type f -empty
```

Comment:
Comment above says "Delete", but `rm` part is missing.
Maybe add `-name "$POLICYLIST_FILENAME"` based on the comment above, this applies to all empty files.

Reply:
Ops, I fogot the `-delete` argument.